CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 2.3.2 (Next)
 
+* [#32](https://github.com/alexa-js/alexa-app-server/pull/32): group of bug fixes and more testing/coverage - [@tejashah88](https://github.com/tejashah88)
```

Comment:
Lets expand this to be specific to be helpful to people reading the CHANGELOG, just list actual fixes on separate lines? Also start with a Capital letter and end with a period ;) `<nitpick />`